### PR TITLE
Migrate StaticCountdownTimer from Reaction

### PR DESCRIPTION
**Pull request by dblandin**
Migrate `<StaticCountdownTimer>` component from Reaction.

**Note**: This ports over a dependency from Reaction as well: `moment-timezone`. We eventually want to switch to [Luxon](https://moment.github.io/luxon/) but that'll first have to go through the [new dependency RFC process](https://github.com/artsy/README/blob/master/playbooks/dependencies.md#a-rubric-on-whether-a-dependency-needs-an-rfc).

2019-01-17-175440_1386x1731_scrot

### CODE REVIEW

File: src/elements/StaticCountdownTimer/StaticCountdownTimer.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import moment from "moment-timezone"
```
Comment (damassi) [inline src/elements/StaticCountdownTimer/StaticCountdownTimer.tsx]: This is such a heavy dependency :(

Comment (damassi) [reply src/elements/StaticCountdownTimer/StaticCountdownTimer.tsx]: Would you mind adding a FIXME here?

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit 326fa8c922335e893a29326147e8c4116488fb5e

https://deploy-preview-205--artsy-palette.netlify.com

**Comment by damassi** [comment]
👍

**Pull request merged**